project/src/main/java/com/google/sps/servlets/LoginStatusServlet.java

```diff
@@ -8,39 +8,38 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns the login status of the user. */
+/** Servlet that returns whether user is logged in and email of the user. */
 @WebServlet("/login-status")
 public class LoginStatusServlet extends HttpServlet {
+  /**
+   * Writes whether user is logged in and email of user if the user is logged in to response.
+   *
+   * @param request servlet request
+   * @param response servlet response
+   * @throws IOException if Input/Output error occurs
+   */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("application/json;");
 
-    // Get the current login status.
     LoginStatus status;
     if (UserServiceFactory.getUserService().isUserLoggedIn()) {
-      status = LoginStatus.getLoggedInInstance();
+      status =
+          new LoginStatus(true, UserServiceFactory.getUserService().getCurrentUser().getEmail());
     } else {
-      status = LoginStatus.getNotLoggedInInstance();
+      status = new LoginStatus(false, "");
```

Comment:
Decided not to use optional here for email because LoginStatus object will be converted into JSON.